drivers/hmis/app/models/hmis/ce/filtered_candidates_query.rb

```diff
@@ -0,0 +1,104 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Hmis::Ce
+  class FilteredCandidatesQuery
+    include ::Hmis::Concerns::HmisArelHelper
```

Comment:
This is fine as-is, we use this mix-in a lot. However there's an alternative which uses composition instead.

This is better imo because it avoids  including all the helper methods into the classes nam
```ruby 
  def arel = Hmis::ArelHelper.instance

  # then 
  def foo
    arel.ug_t[:id].eq(@unit_group_id)
  end 
```